src/compound-assignment-private/default/data-property.template

```diff
@@ -0,0 +1,51 @@
+// Copyright 2021 the V8 project authors; 2022 Igalia S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+path: language/expressions/compound-assignment/left-hand-side-private-reference-data-property-
+esid: sec-assignment-operators-runtime-semantics-evaluation
+info: |
+  sec-assignment-operators-runtime-semantics-evaluation
+  AssignmentExpression : LeftHandSideExpression AssignmentOperator AssignmentExpression
+    1. Let _lref_ be the result of evaluating |LeftHandSideExpression|.
+    2. Let _lval_ be ? GetValue(_lref_).
+    ...
+    7. Let _r_ be ApplyStringOrNumericBinaryOperator(_lval_, _opText_, _rval_).
+    8. Perform ? PutValue(_lref_, _r_).
+    9. Return _r_.
+
+  sec-property-accessors-runtime-semantics-evaluation
+  MemberExpression : MemberExpression `.` PrivateIdentifier
+
+    1. Let _baseReference_ be the result of evaluating |MemberExpression|.
+    2. Let _baseValue_ be ? GetValue(_baseReference_).
+    3. Let _fieldNameString_ be the StringValue of |PrivateIdentifier|.
+    4. Return ! MakePrivateReference(_baseValue_, _fieldNameString_).
+
+  PutValue (V, W)
+    ...
+    5.b. If IsPrivateReference(_V_) is *true*, then
+      i. Return ? PrivateSet(_baseObj_, _V_.[[ReferencedName]], _W_).
+
+  PrivateSet (O, P, value)
+    ...
+    3. If _entry_.[[Kind]] is ~field~, then
+      a. Set _entry_.[[Value]] to _value_.
+
+name: to a field
+features: [class-fields-private]
+---*/
+
+class C {
+  #field = /*{lhs}*/;
```

Comment:
Me not saying anything about this...
![](https://media3.giphy.com/media/l4Jz3fRRdJsd7HStG/giphy.gif)